src/jit/importer.cpp

```diff
@@ -13351,7 +13351,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         }
                     }
 
-                    if (prefixFlags & PREFIX_UNALIGNED)
```

Comment:
This code just sets the `GTF_IND_UNALIGNED` if it is not using a helper.

Not sure what indentation is broken.  clang-format was happy with it.  Perhaps you missed the discontinuityin the patch line numbers?